v21.1/information-schema.md

```diff
@@ -231,6 +441,12 @@ Column | Description
 `increment` | The value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.
 `cycle_option` | Currently, all sequences are set to `NO CYCLE` and the sequence will not wrap.
 
+### session_variables
+
+<span class="version-tag">New in v21.1:</span> `session_variables` contains information about the [session variable settings](set-vars.html) for your session. The `session_variables` view exposes the same information as the [`SHOW (session settings)`](show-vars.html) statement.
```

Comment:
The `session_variables` view exposes wording may be a little bit confusing since it's a table and not a view.

I would say something mention something like 
`session_variables` has two columns `variable` and `value` where the `value` corresponds to the output of `SHOW variable`.